libraries/FunctionalInterrupt/src/FunctionalInterrupt.h

```diff
@@ -0,0 +1,18 @@
+#ifndef FUNCTIONALINTERRUPT_H
+#define FUNCTIONALINTERRUPT_H
+
+#include <functional>
+
+// Structures for communication
+
+struct InterruptInfo
```

Comment:
I thought from the changes in wiring that this was dropped. Why is it available only for scheduled interrupts, and not for normal interrupts?
I'm thinking maybe it should be implemented in a better way, e.g.: read the values as early as possible in the ISR.

Reply:
Please check my comment above.